core/arch/arm/kernel/stmm_sp.c

```diff
@@ -849,17 +846,26 @@ static bool spm_handle_svc(struct thread_svc_regs *regs)
 	case FFA_VERSION:
 		DMSG("Received FFA version");
 		*a0 = MAKE_FFA_VERSION(FFA_VERSION_MAJOR, FFA_VERSION_MINOR);
-		return true;
```

Comment:
Why not just return here? For this branch of this function it end here. Same for the other cases below.